src/main/java/Mentos/components/ActionTaskIndexTuple.java

```diff
@@ -0,0 +1,28 @@
+package Mentos.components;
+
+import Mentos.task.Task;
+
+public class ActionTaskIndexTuple {
```

Comment:
At first glance, the class name here seems slightly complicated to me! Perhaps a header comment could be included here to explain what the class does? Alternatively, you can also consider using a shorter and simpler name!